app/src/cli.c

```diff
@@ -2493,6 +2499,9 @@ parse_args_with_getopt(struct scrcpy_cli_args *args, int argc, char *argv[],
             case OPT_CAMERA_HIGH_SPEED:
                 opts->camera_high_speed = true;
                 break;
+            case OPT_ROOT:
```

Comment:
Nit: move this under `OPT_NO_WINDOW` to preserve order of the enum at the beginning of the file

Reply:
Will be flipped around per your suggestion in a fixup commit.